### DISCUSSION TITLE
Icons Request: High-Heeled Shoe (icon-stiletto, icon-heel)

### DISCUSSION THREAD

**Discussion by marcofrasson** [FortAwesome/Font-Awesome#21220] [Icon Requests]
I have imported a store that will sell running shoes and shoes. I created these two icons in AI and SVG format for deployment.

AI https://drive.google.com/file/d/0B6yWrleAQMXWM0dRbUNyaUN3VE0/
SVG https://drive.google.com/file/d/0B6yWrleAQMXWeHdUM0JoV1ItNkk/

![image](https://cloud.githubusercontent.com/assets/556268/3272735/963a6c10-f31b-11e3-9c55-a00447e3907b.png)

**Comment by ShamanFromSiberia** [comment]
+1 to the idea, but not realization.

**Comment by christianyfrau** [comment]
+1 to this idea but we need a more 'classy' icon.  See attached.  This icon is more in line with the other icons in Font Awesome.
![14580518-women-heel-shoes-silhouettes](https://cloud.githubusercontent.com/assets/9102524/4572913/2f8aa8ae-4f8c-11e4-821a-d16a42e545f7.png)

**Comment by AndyArcherKG** [comment]
:+1:  this idea for sure.
but agree the icon probably needs some more work.

**Comment by Elias87** [comment]
+1

**Comment by multiwins** [comment]
Hey guys! When you include this SVG into FA? Now FA doesn't have icons about shoes! Please include this :)

**Comment by tagliala** [comment]
@AngelWeb at the moment this icon is not planned, but you can create a custom build. Take a look here: https://github.com/FortAwesome/Font-Awesome/wiki/Customize-Font-Awesome

**Comment by jbnv** [comment]
+1.

**Comment by ShrawanPatel** [comment]
+1

**Comment by simonfranz** [comment]
+1

**Comment by AndreLZGava** [comment]
Hello guys

I love font awesome, but some icons that we need take a lot of time to be added to font awesome, so I decided to create an extenssion to make it easy , if you would like to use them , you can download this at https://github.com/AndreLGava/font-awesome-extenssion. If there are any problem please create an issue on the GitHub project

**Comment by saivionoutsourcingservices** [comment]
**Please guys tell me fast, what is the html icon code for cobbler or shoes? It's very urgent....**

**Comment by naomi789** [comment]
+1

**Comment by aethior** [comment]
*Related Unicode Emoji* #222
👠 [U+1F460](https://emojipedia.org/emoji/%F0%9F%91%A0/) High-Heeled Shoe
Also known as `High Heels` and `Stiletto`.

Related requests:
👠 High-Heeled Shoe https://github.com/FortAwesome/Font-Awesome/issues/3473
👟 Running Shoe https://github.com/FortAwesome/Font-Awesome/issues/3704
👞 Man’s Shoe https://github.com/FortAwesome/Font-Awesome/issues/6031
🩰 Ballet Shoes https://github.com/FortAwesome/Font-Awesome/issues/7563

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
